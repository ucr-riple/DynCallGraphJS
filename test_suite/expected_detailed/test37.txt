{
    'global (test37.js:6:1:8:3)': [
        'call (Native)(test37.js:6:1:8:3)'
    ],
    'call (Native) (test37.js:6:1:8:3)': [
        'anon (test37.js:6:35:8:2)'
    ],
    'anon (test37.js:7:10:7:15)': [
        'anon (test37.js:2:3:4:4)'
    ]
}