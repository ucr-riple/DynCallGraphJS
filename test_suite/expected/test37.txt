{ '(test37.js:6:1:8:3)': [ 'call (Native)' ],
  'call (Native)':
   [ '(test37.js:6:35:8:2)' ],
  '(test37.js:7:10:7:15)':
   [ '(test37.js:2:3:4:4)' ] }